woocommerce/woocommerce-ios
# Analytics Hub: Introduce time range picker

Part of #8146

Why
==========
 We need a way to change the stats time range selection inside the Analytics Hub itself. Subsequent PRs shall start introducing the remaining ranges from the issue description.

How
==========
Introduce a date range UI selection list view for the currently available ranges and make the Time range card clickable with the suggested chevron hint. Makes the `SelectionType` a published value that causes a reaction to the currently generated Time ranges every time the selection list changes. Subsequent PRs will remove the `SelectionType` enum from the `AnalyticsHubTimeRangeGenerator` to the `AnalyticsHubViewModel`, since that enum is becoming a UI control value instead of a time business logic.

Screen Capture
==========
https://user-images.githubusercontent.com/5920403/204597656-1377af29-3031-4684-b4a2-f67684dd9fb7.mp4

How to Test
==========
1. Launch the app and click the `See More button in the My Store charts section.
2. Verify that the currently selected Date Range corresponds to the My Store stats entry point
3. Hit the Date Range section and verify that a view with the range picker is presented
4. Change the selected range, verify that the displayed stats correspond to what's displayed inside Woo Core

Update release notes:
